auth-api/src/auth_api/services/task.py

```diff
@@ -55,21 +60,54 @@ def as_dict(self):
         return obj
 
     @staticmethod
-    def create_task(task_model: TaskModel):
+    def create_task(org_info: dict):
         """Create a new task record."""
-        current_app.logger.debug('<create_staff_task ')
-
+        current_app.logger.debug('<create_task ')
+        task_model = TaskModel(name=org_info.get('name'),
```

Comment:
this should be expandable to other kind of relations as well rite? like affidavit approval..Not just org approval